examples/modules/plus-number/README.md

```diff
@@ -76,7 +76,7 @@ namespace demo {
 
         /// <summary> Declare persistent constructor to create PlusNumber instance. </summary>
         /// <remarks> Napa creates persistent constructor at each isolate while node.js creates the static instance. </remarks>
-        NAPA_DECLARE_PERSISTENT_CONSTRUCTOR
+        NAPA_DECLARE_PERSISTENT_CONSTRUCTOR;
```

Comment:
This looks strange.

How about modify the macro to have NAPA_DECLARE_PERSISTENT_CONSTRUCTOR()?

Reply:
That will break the build since macro NAPA_DECLARE_PERSISTENT_CONSTRUCTOR does not have any parameter

Reply:
Any comments update?

Current summary:
- `NAPA_DECLARE_PERSISTENT_CONSTRUCTOR;` is always better than `NAPA_DECLARE_PERSISTENT_CONSTRUCTOR` in source code
- `NAPA_DECLARE_PERSISTENT_CONSTRUCTOR();` looks great but it will break the build under current definition. To allow this usage the definition must be something like:
    ```c
    #ifdef BUILDING_NAPA_EXTENSION
    #define NAPA_DECLARE_PERSISTENT_CONSTRUCTOR(dummy)
    #else
    #define NAPA_DECLARE_PERSISTENT_CONSTRUCTOR(dummy) \
        static v8::Persistent<v8::Function> _constructor
    #endif             
    ```
    Is it a good idea to introduce the `dummy` here?

Reply:
@daiyip @helloshuangzi this is a trival change but it affect the code for napa native addon. Would you please draw some comments on this?